catalog_client/agents/node_agent.py

```diff
@@ -20,4 +22,6 @@ def collect(self):
             env=environ,
         ).addCallback(
             lambda version: version.strip()
+        ).addErrback(
+            lambda _ : "Unknown Version"
```

Comment:
on second thought, is there anyway we _can_ get the flocker version that doesn't use flocker-diagnostics, unknown version is almost as bad as not reporting it IMO